### DISCUSSION TITLE
Repo for Language Tour?

### DISCUSSION THREAD

**Discussion by abellgithub** [hylo-lang/hylo#793] [Q&A] [answered]
I've noticed a typo in the Language Tour. Should I make a PR against it? If so, an someone point me to the repo?

**Comment by kyouko-taiga** [comment]
Hi,

We have not configured PR for the language tour yet (it's hosted on GitBook). Put please open an issue on https://github.com/val-lang/val-lang.github.io in the meantime.

Thanks a lot.

**Reply by hawkinsw** [reply]
I would love to be able to help with the Tour -- having a repo for it would be awesome! I can imagine that accepting PRs to fix typos etc would probably be easier for you, too, instead of having issues pile up for typos. 

The Tour is great! Thank you *so* much!

**Reply by kyouko-taiga** [reply]
Thank you! I'm new to GitBook so I'll see what can be done.

**Reply by kyouko-taiga** [reply]
There you are: https://github.com/val-lang/GitBooks/tree/main/Tour

**Comment by hawkinsw** [comment]
Awesome!! Thanks!!

**Comment by tothambrus11** [comment, marked as answer]
Update: now our website (including the language tour) lives in the following repo: https://github.com/hylo-lang/hylo-lang.github.io, and it's live at https://hylo-lang.org.
